File: sampletargetj/src/main/java/sampletargetj/Bean1.java
package sampletargetj;

import java.util.List;
import java.util.stream.Collectors;

//@Autowire
public class Bean1 {

    final IService1 service1;
    public Bean1(IService1 service1) {
        this.service1=service1;
    }

    public String greet(List<String> names)  {
        return names.stream().map(it-> service1.sayHello(it) ).collect(Collectors.joining(","));
    }

    public String officialGreet(List<Pair<String, String>> names)  {
        return names.stream().map ( it-> service1.sayHello(it.first, it.second) ).collect(Collectors.joining(","));
    }
}

File: sampletarget/src/main/kotlin/targetsample/Bean1.kt
package targetsample

class Bean1(val service1: IService1) {
    fun greet(names: List<String>) : String {
        return names.map { service1.sayHello(it) }.joinToString(",")
    }
    fun officialGreet(names: List<Pair<String, String>>) : String {
        return names.map { service1.sayHello(it.first, it.second) }.joinToString(",")
    }
}